api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -390,7 +390,8 @@ message FilterStateRule {
 
   // A map of string keys to requirements. The string key is the string value
   // in the FilterState with the name specified in the *name* field above.
-  map<string, JwtRequirement> requires = 3;
+  map<string, JwtRequirement>
+  requires = 3;
```

Comment:
I wonder if this change was accidental. `FORCE_PROTO_FORMAT=yes ./tools/proto_format/proto_format.sh check` complains about this. 
(same applies to corresponding v3/config.proto)